src/org/ods/component/ScanWithAquaStage.groovy

```diff
@@ -199,13 +215,28 @@ class ScanWithAquaStage extends Stage {
                 [ title: "Messages", value: prepareMessageToBitbucket(messages), ]
             ])
         }
+        if (actionableVulnerabilities?.size() > 0) {
```

Comment:
Why not add a new message per actionableVulnerabilitie like: title: "Explotable vulnerability", value: Information about the vulnerability (using prepareMessageToBitbucket)

Reply:
We need this specific entry to be present in the data structure as a marker: {
            "title": "Blocking",
            "type": "TEXT",
            "value": "Yes"
        }